http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpSamDAO.scala

```diff
@@ -179,6 +179,37 @@ class HttpSamDAO[F[_]](httpClient: Client[F],
         }
     } yield ()
 
+  override def createResourceV2[R](resource: R,
+                                   creatorEmail: WorkbenchEmail,
```

Comment:
It looks like `creatorEmail` is not used at all and `cloudContext` is just used for logging. Thinking maybe we log `"${sr.resourceType.asString}/${sr.resourceIdAsString(resource)}"` instead and just remove these params?